rleplus/internal/bitvector_test.go

```diff
@@ -0,0 +1,124 @@
+package bitvector_test
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/filecoin-project/go-filecoin/rleplus/internal"
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	"gotest.tools/assert"
+)
+
+func assertBitVector(t *testing.T, expected []byte, actual bitvector.BitVector) {
```

Comment:
might be worth mentioning in comment that `expected`'s elements should only ever be `0` and `1` bytes.  Perhaps you could call it `expectedBits` too.